server/database/setup.sql

```diff
@@ -0,0 +1,40 @@
+DROP DATABASE IF EXISTS garager;
+CREATE DATABASE garager;
+
+\c garager;
+
+DROP TABLE IF EXISTS users;
+CREATE TABLE users (
+  user_id SERIAL PRIMARY KEY
+);
+
+
+DROP TABLE IF EXISTS items;
+CREATE TABLE items (
+  item_id SERIAL PRIMARY KEY,
+  user_id INTEGER NOT NULL DEFAULT NULL,
+  title TEXT NOT NULL DEFAULT NULL,
+  category TEXT NOT NULL DEFAULT NULL,
+  brand TEXT NOT NULL DEFAULT NULL,
+  model TEXT NOT NULL DEFAULT NULL,
+  itemDescription TEXT NOT NULL DEFAULT NULL,
```

Comment:
I forgot where the team landed on which of parts were required. I thought maybe brand/model was optional. But I guess if you're handling posting of items, can you pass in an empty string if the user doesn't input anything?

Reply:
I'm thinking to put as N/A as default. I don't want GET to get mix match result for some empty column....